notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2988 @@
+{
```

Comment:
<p>Wow. For what it's worth, this is an excellent plot for the amount of fudging we've done.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='96'/>